src/Tour.js

```diff
@@ -360,14 +360,15 @@ class Tour extends Component {
       nextStep,
       prevStep,
       disableKeyboardNavigation,
+      showCloseButton,
     } = this.props
     e.stopPropagation()
 
     if (disableKeyboardNavigation) {
       return
     }
 
-    if (e.keyCode === 27) {
+    if (e.keyCode === 27 && showCloseButton) {
```

Comment:
Instead of mix two (in some way) unrelated `props`, I mean, an UI element (like `x` button) and a behavior like close the Tour by pressing <kbd>esc</kbd> we could do something like:
- [ ] Accept an `Object` when passing `disableKeyboardNavigation` prop which includes
  - `navigation`, `close`, etc
- [ ] Check in the conditions ` if (e.keyCode === … && …)` if was disabled this specific feature

Let me know what do you think,

Reply:
I've updated it so disableKeyboardNavigation can either be boolean or array of disabled keys ['esc', 'right', 'left']